snuba/admin/auth.py

```diff
@@ -33,10 +37,41 @@ def authorize_request() -> AdminUser:
     return _set_roles(provider())
 
 
+def _is_member_of_group(user: AdminUser, group: str) -> bool:
+    # TODO this will be handled by gcp identity once we have the service account
+    # https://cloud.google.com/identity/docs/how-to/query-memberships#searching_for_all_memberships_in_a_group
+    return False
+
+
+def get_iam_roles_from_file(user: AdminUser) -> Sequence[str]:
+    iam_roles = []
+    try:
+        with open(settings.ADMIN_IAM_POLICY_FILE, "r") as policy_file:
+            policy = json.load(policy_file)
```

Comment:
should we handle invalid json errors as well?

Reply:
Not sure about this. My feeling is that if someone put a policy file and it's not proper json its probably better to fail cause that means something is seriously wrong. I wouldn't expect in normal situations for someone to put invalid json.